### PR TITLE
Create aakarshan_haiku.md

### PR DESCRIPTION
**Pull request by aakarshank**
first Hacktoberfest pull request

### CODE REVIEW

File: _haikus/aakarshan_haiku.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Aakarshan Haiku
+author:aakarshank
```
Comment (ltagliaferri) [inline _haikus/aakarshan_haiku.md:4]:
```suggestion
author: aakarshank
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Can you fix the formatting issue in the header please? Thank you

**Review by aakarshank — COMMENTED** [review]
Changed the formatting

File: _haikus/aakarshan_haiku.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout:haiku
+title:Aakarshan Haiku
```
Comment (ltagliaferri) [inline _haikus/aakarshan_haiku.md:3]:
```suggestion
title: Aakarshan Haiku
```

File: _haikus/aakarshan_haiku.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+layout:haiku
```
Comment (ltagliaferri) [inline _haikus/aakarshan_haiku.md:2]:
```suggestion
layout: haiku
```

**Pull request merged**